components/reference-data/src/main/java/org/datacleaner/beans/filter/DictionaryFilter.java

```diff
@@ -47,20 +52,39 @@
     @Configured
     Dictionary dictionary;
 
+    @Provided
+    DataCleanerConfiguration configuration;
+
+    private DictionaryConnection dictionaryConnection;
+
     public DictionaryFilter() {
     }
 
-    public DictionaryFilter(InputColumn<String> column, Dictionary dictionary) {
+    public DictionaryFilter(InputColumn<String> column, Dictionary dictionary, DataCleanerConfiguration configuration) {
```

Comment:
That whole constructor is only used in test. It's just to make it easy to identify if a unittest has satisfied all the properties or not.